RodiSales87/pta-squad-drigo
# Feat/card

### What I did

- Created the Card component to display dynamic information such as game name, event date, time, participants, and meeting platform.
- Implemented conditional logic to change the card color based on the event's state (e.g., creator, expired event, participants below or at the limit).
- Prepared the component to easily integrate with the backend by using dynamic data for the card's content.

### How to test

1. Open the terminal and go to the folder of this project
2. In terminal:

- `git checkout develop`
- `git pull`
- `git checkout feat/card`
- `git pull origin feat/card`
- `yarn install`
- `yarn dev`

3. In your browser go to
4. Check if the card displays the correct information, such as game name, event date, time, participants, and meeting platform.
5. Verify that the card color changes correctly based on different states:

- Event creator
- Expired event
- Participants below or at the limit
